app/services/preserved_object_handler.rb

```diff
@@ -79,28 +79,28 @@ def check_existence
           if incoming_version == pres_copy.version
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_copy.class.name)
             handler_results.add_result(PreservedObjectHandlerResults::VERSION_MATCHES, pres_object.class.name)
-            update_pc_validation_timestamps(pres_copy)
-            update_db_object(pres_copy)
-          else
-            version_comp_result = if incoming_version > pres_copy.version
-                                    PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT
-                                  else
-                                    PreservedObjectHandlerResults::ARG_VERSION_LESS_THAN_DB_OBJECT
-                                  end
-            handler_results.add_result(version_comp_result, pres_copy.class.name)
-            handler_results.add_result(version_comp_result, pres_object.class.name)
+          elsif incoming_version > pres_copy.version
+            handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_copy.class.name)
+            handler_results.add_result(PreservedObjectHandlerResults::ARG_VERSION_GREATER_THAN_DB_OBJECT, pres_object.class.name)
             if moab_validation_errors.empty?
-              update_preserved_copy_version_etc(pres_copy, incoming_version, incoming_size, true)
+              update_preserved_copy_version_etc(pres_copy, incoming_version, incoming_size, false)
```

Comment:
should this stay `true` since we looked at the validation errors and determined there were none?

Reply:
I made it false because I moved the timestamp updates to outside the if block, and that's all that "true" as an arg is doing:  telling the method to update validation timestamps.  "true" is more correct, but "false" avoids double updates.